Error: Cannot find module './invariant'

Hi,
sorry to bother you but when I run

```javascript
npm install
```

I've got
```javascript
> flux@2.0.2 prepublish /home/user
> gulp publish

[16:12:05] Using gulpfile ~/flux/source/Gulpfile.js
[16:12:05] Starting 'clean'...
[16:12:05] Starting 'lib'...
[16:12:05] Starting 'browserify'...
[16:12:06] Finished 'clean' after 136 ms

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: Cannot find module './invariant'
```

 can you help me, please ?

Best regards.

--- a/Gulpfile.js
+++ b/Gulpfile.js
@@ -4,6 +4,7 @@ var gReplace = require('gulp-replace');
 var browserify = require('browserify');
 var source = require('vinyl-source-stream');
 var del = require('del');
+var runSequence = require('run-sequence');
 
 var browserifyConfig = {
   entries: ['./index.js'],
@@ -22,12 +23,17 @@ gulp.task('lib', function() {
 
 });
 
-gulp.task('browserify', function() {
+gulp.task('browserify', ['lib'], function() {
   return browserify(browserifyConfig)
           .bundle()
           .pipe(source('Flux.js'))
           .pipe(gulp.dest('./dist/'))
 });
 
-gulp.task('publish', ['clean', 'default']);
-gulp.task('default', ['lib', 'browserify']);
+gulp.task('build', ['lib', 'browserify']);
+
+gulp.task('publish', function(cb) {
+  runSequence('clean', 'build', cb);
+});
+
+gulp.task('default', ['build']);

--- a/package.json
+++ b/package.json
@@ -44,6 +44,7 @@
     "gulp-replace": "^0.4.0",
     "jest-cli": "~0.1.17",
     "react-tools": "^0.12.0",
+    "run-sequence": "^1.0.2",
     "vinyl-source-stream": "^0.1.1"
   }
 }

